Music-Player-App/readme.MD

```diff
@@ -0,0 +1,43 @@
+# Music-Player
+=======
+# Music Player App Using Pygame and Mutagen
```

Comment:
After this heading, make every subheading in level two (##)

Reply:
Done...!!!